### ISSUE TITLE
Accessing values from standalone config file

### ISSUE THREAD

**Issue by jgoclawski** [jasonmit/ember-intl#319]
With config options in "config/environment.js" file we could easily access them in our Ember code.

Real-life scenario - setting fallback runtime locale:

```
// app/routes/application.js
import Ember from 'ember';
import settings from '[project-name]/config/environment';

export default Ember.Route.extend({
  beforeModel() {
    ...
    [get browser locales, try to match them to available locales]
    ...
    if (userLocaleNotMatchedToAvailable) {
      this.get('intl').setLocale(settings.baseLocale);
    }
  }
});
```

How can we achieve the same result with config options in a standalone file? Is it still possible?

**Comment by jasonmit** [comment]
config/environment is still supported but deprecated, if you need to hang properties to evaluate at runtime feel free to keep using it.  In the future I'll need to solve this before its removed.

**Comment by jgoclawski** [comment]
Ok, thanks! In that case, we'll keep using the old-style config for now. Feel free to close the issue :)

**Comment by jasonmit** [comment]
I will keep open for tracking to make sure it's addressed as part of 3.0

Thanks again

**Comment by jasonmit** [comment]
When https://github.com/jasonmit/ember-intl/pull/348 is versioned you'll have access to the contents of ember-intl.js from the traditional app/config/environment
